assets/css/site/header.css

```diff
@@ -71,6 +71,10 @@
     height: 24px;
 }
 
+.menu-item-more svg {
+    fill: currentColor;
```

Comment:
We have a selector `.menu-item-more .icon` for the icon [here](https://github.com/TryGhost/Dawn/blob/cea820bd3f089649d342902c528f01b9b2ba540c/assets/css/site/header.css#L69). Would you be able to move this rule there? 🙂